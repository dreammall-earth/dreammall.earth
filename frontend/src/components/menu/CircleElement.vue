<template>
  <div class="circle rounded-circle border-sm text-font pa-2 d-flex justify-center align-center">
    <slot />
  </div>
</template>

<style scoped lang="scss">
.circle {
  width: var(--menu-icon-height);
  height: var(--menu-icon-height);
  background: var(--v-icon-background) !important;
}
</style>
