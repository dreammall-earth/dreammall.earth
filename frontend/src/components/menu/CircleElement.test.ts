import { mount } from '@vue/test-utils'
import { describe, it, expect, beforeEach } from 'vitest'

import CircleElement from './CircleElement.vue'

describe('CircleElement', () => {
  const Wrapper = () => {
    return mount(CircleElement)
  }
  let wrapper: ReturnType<typeof Wrapper>

  beforeEach(() => {
    wrapper = Wrapper()
  })

  it('renders', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
