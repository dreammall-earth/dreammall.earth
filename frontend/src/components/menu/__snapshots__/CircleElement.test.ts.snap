// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`CircleElement > renders 1`] = `
<div
  class="circle rounded-circle border-sm text-icon pa-2 d-flex justify-center align-center"
  data-v-39e8094e=""
>
  
  
</div>
`;
